src/main.rs

```diff
@@ -111,12 +112,12 @@ fn main() {
         Some("send") if args.len() >= 3 => {
             let addr = &args[1];
             let fnames = &args[2..];
-            main_send(addr, fnames).expect("Failed to send.");
+            main_send(addr, fnames, WIRE_PROTO_VERSION).expect("Failed to send.");
         }
         Some("recv") if args.len() == 3 => {
             let addr = &args[1];
             let n_conn = &args[2];
-            main_recv(addr, n_conn).expect("Failed to receive.");
+            main_recv(addr, n_conn, true, WIRE_PROTO_VERSION).expect("Failed to receive.");
```

Comment:
```suggestion
            main_recv(addr, n_conn, WriteMode::AskConfirm, WIRE_PROTO_VERSION).expect("Failed to receive.");
```
This is a case of the [boolean parameter trap](https://wiki.qt.io/API_Design_Principles#The_Boolean_Parameter_Trap). On the call site, a reader might wonder what `recv(true)` means. Does it mean that `recv(false)` stops receiving?

One way to avoid this is to give the value a name, e.g.:
```rust
let ask_confirm = true;
main_recv(addr, n_conn, ask_confirm, WIRE_PROTO_VERSION);
```
A different solution that’s a bit more verbose but also prevents the issue for future callers is to define an enum, something like
```rust
enum WriteMode {
    /// Display the plan to the user and ask for confirmation before writing to disk.
    AskConfirm,
    /// Write to disk unconditionally, overwriting any files that might exist.
    Overwrite,
}